extensions/wikia/ImageReview/maintenance/ImageReviewDeleteInvalidEntries.php

```diff
@@ -1,19 +1,32 @@
 <?php
 
 require_once __DIR__ . '/../../../../maintenance/Maintenance.php';
+require_once __DIR__ . '/../../../../includes/wikia/nirvana/WikiaObject.class.php';
+require_once __DIR__ . '/../../../../includes/wikia/nirvana/WikiaModel.class.php';
+require_once __DIR__ . '/../../../../lib/Wikia/src/Logger/Loggable.php';
+require_once __DIR__ . '/../../../../includes/wikia/tasks/Tasks/BaseTask.class.php';
+require_once __DIR__ . '/../ImageReviewHelperBase.class.php';
+require_once __DIR__ . '/../ImageReviewHelper.class.php';
+require_once __DIR__ . '/../ImageReviewTask.class.php';
```

Comment:
None of these added requires should be needed, only the Maintenance.php import. The rest of the classes will be autoloaded when used.